src/main/java/com/teamc/fms/service/TimeSlotService.java

```diff
@@ -0,0 +1,22 @@
+package com.teamc.fms.service;
+
+import com.teamc.fms.dto.TimeSlotDto;
+import com.teamc.fms.entity.TimeSlot;
+import com.teamc.fms.model.CreateTimeSlotCommand;
+
+import java.util.List;
+
+public interface TimeSlotService {
+
+    TimeSlot updateTimeSlot(String id, CreateTimeSlotCommand command);
+
```

Comment:
As discussed above, please use UpdateTimeslotCommand instead of CreateTimeSlotCommand